packages/nx/src/command-line/release/utils/github.ts

```diff
@@ -227,9 +301,9 @@ function githubNewReleaseURL(
   config: GithubRequestConfig,
   release: { version: string; body: string }
 ) {
-  return `https://github.com/${config.repo}/releases/new?tag=v${
+  return `https://github.com/${config.repo}/releases/new?tag=${
```

Comment:
These days release.version will already be formatted appropriately based on the user's defined `releaseTagPattern` (which may or may not include a `v`, it's up to them)